### DISCUSSION TITLE
Does not work on ports other than 9091

### DISCUSSION THREAD

**Discussion by Shesakillatwo** [authelia/authelia#4822] [Q&A] [answered]
### Version

v4.37.5

### Deployment Method

Docker

### Reverse Proxy

SWAG

### Description

Attempting to install and configure authelia to run on a port different than 9091 as I have another device on my network that talks to my unraid server on that port.  I tried ports 9098 and 9092 and niether port worked.  After a full setup I was unable to reach the local network authentication page for authelia with my browser until I set the port back to 9091.  Once set on 9091 it worked fine and connected.

Regardless of which port I used the Authelia logs indicated that it started correctly though I could not use it.

### Reproduction

install using the ibracorp video and use a different port.

### Expectations

I would like it to work on a different port otherwise I cannot use this application

### Configuration (Authelia)

```yaml
Standard config from ibracorp docs pages shown in recent video.  The only difference is when the port was change to other than 9091.  WHen it was at 9092 or 9098 authelia would not connect.  When changed back to 9091 it would connect.
```

### Logs (Authelia)

```shell
time="2023-01-23T14:52:31-05:00" level=info msg="Storage schema is being checked for updates"
time="2023-01-23T14:52:31-05:00" level=info msg="Storage schema is already up to date"
time="2023-01-23T14:52:31-05:00" level=debug msg="notification provider: startup check skipped as it is disabled"
time="2023-01-23T14:52:31-05:00" level=debug msg="The NTP startup check was skipped due to there being no configured 2FA access control rules"
time="2023-01-23T14:52:31-05:00" level=info msg="Initializing server for non-TLS connections on '[::]:9098' path '/' and '/authelia'"
```

### Logs (Proxy / Application)

```shell
Not applicable as this does not even work within my network on local IP address.
```

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)
- [X] This is a bug report and not a support request
- [X] I have read the security policy and this bug report is not a security issue or security related issue
- [X] I have either included the complete configuration file or I am sure it's unrelated to the configuration
- [X] I have provided all of the required information in full with the only alteration being reasonable sanitization in accordance with the [Troubleshooting Sanitization](https://www.authelia.com/r/sanitize) reference guide
- [X] I have checked for related proxy or application logs and included them if available
- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment]
Why didn't you complete the form and why didn't you actually explain your issue? Saying it doesn't work is not informative.

**Comment by james-d-elliott** [comment]
Actually, there is no way this issue exists as it's tested, this is a support question due to a config issue on your end and should be a discussion. Please provide your docker compose, proxy configs, and authelia configs.

**Comment by james-d-elliott** [comment, marked as answer]
The answer to this question ended up being the user was trying to map the docker host port to the 9091 container port, but had changed the container port in the configuration.
